tests/unit/services/sl-behavior-test.js

```diff
@@ -218,114 +152,65 @@ test( 'isAble() requires at least two parameters to be provided', function( asse
 });
 
 test( 'isAble() requires first parameter to be a String', function( assert ) {
-    // Empty parameter
-
-    let assertionThrown = false;
-
-    try {
-        BS.isAble();
-    } catch( error ) {
-        assertionThrown = true;
-    }
 
-    assert.ok(
-        assertionThrown,
-        'Parameter was empty'
-    );
-
-    // null parameter
-
-    assertionThrown = false;
-
-    try {
-        BS.isAble( null, 'notUnderTest' );
-    } catch( error ) {
-        assertionThrown = true;
-    }
-
-    assert.ok(
-        assertionThrown,
-        'Parameter was null'
-    );
-
-    // Number parameter
-
-    assertionThrown = false;
-
-    try {
-        BS.isAble( 4, 'notUnderTest' );
-    } catch( error ) {
-        assertionThrown = true;
-    }
+    // Array Parameter
+    let callIsAble = () => BS.isAble( [], 'notUnderTest' );
```

Comment:
For this assignment, and all others in this test, is the second parameter required to be passed since the first parameter is what is being asserted against?  Could possibly be needed for the string test